diff --git a/node_modules/tsdx/dist/createRollupConfig.js b/node_modules/tsdx/dist/createRollupConfig.js
index 0d1c890..ddf1c30 100644
--- a/node_modules/tsdx/dist/createRollupConfig.js
+++ b/node_modules/tsdx/dist/createRollupConfig.js
@@ -34,7 +34,7 @@ async function createRollupConfig(opts) {
         .join('.');
     let tsconfigJSON;
     try {
-        tsconfigJSON = await fs.readJSON(constants_1.paths.tsconfigJson);
+        tsconfigJSON = await fs.readJSON(opts.tsconfig || constants_1.paths.tsconfigJson);
     }
     catch (e) { }
     return {
